modules/47_regipol/regiCarbonPrice/bounds.gms

```diff
@@ -130,6 +130,10 @@ if (cm_EnSecScen_limit gt 0,
     vm_prodPe.up(t,regi,"pegas")$((t.val ge 2025) AND (sameas(regi,"DEU"))) = cm_EnSecScen_limit/pm_conv_TWa_EJ;
 );
 
+*** Fix CES function quantity trajectories to exogenous data if cm_exogDem_scen is activated
```

Comment:
As discussed previously, fixing the ces quantity directly like you do here is not a good practice to be used in any scenario definition as it affects not only this CES tree node, but also the marginal rate of substitution of any nodes connected to it. 
If you really want to push this to the main REMIND code I would add a warning in every line regarding that you call this switch so it is clear to any user that this switch should only be used if you are aware of the consequences.

Reply:
I think we had agreed that Felix should push this to the trunk now. At a later point in time, it would be interesting to compare runs using the two approaches - my bet would be that the marginals in the rest of the CES tree are the same.

Reply:
This is what I added to the description. I think it should address your point. If the CES function is forced away from the optimal point, then the prices of the CES nodes do not necessarily align with the price signal from the energy system anymore. 
```
*** cm_exogDem_scen
*** switch to fix FE or ES demand represented in CES function to trajectories 
*** from exgenous sources (not EDGE models) given in file p47_exogDemScen.
*** This switch fixes demand without recalibration of REMIND CES parameters.
*** This should be kept in mind when comparing those runs to baseline runs without fixing
*** as the fixing shifts the CES function away from its optimal point based on the CES parameters used.
```

Reply:
I read the comment the first time I reviewed the PR.
I ask you to please add an extra line to it saying something like:
```
*** warning: the formulation fixing CES quantity nodes in scenarios should be used with care and parsimony. Price and tax induced solutions are preferable from the REMIND formulation perspective, and consequences of fixing CES tree nodes directly still require further investigation. 
```

Reply:
Ok, added